test/ziggurat/mapper_test.clj

```diff
@@ -201,6 +201,22 @@
               (is (= message-from-mq expected-message)))
             (is @unsuccessfully-processed?)))))
 
+    (testing "message process shoul˚d be unsuccessful and be pushed to dlq"
```

Comment:
testing block title should change: `message should be published to dead-letter if handler returns :dead-letter keyword`